.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Please remove the blank line at the top.

Reply:
done